modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -175,6 +177,31 @@ Method Label Options Color:: Specifies the OPTIONS method label color.
 Method Label Head Background Color:: Specifies the HEAD method label background color.
 Method Label Head Color:: Specifies the HEAD method label color.
 
+If an *API Console* navigation item such as an API endpoint or method has a long name, it wraps around to a second line. By default, the line breaks at the end of a word. To configure the line break behavior:
+
+. In the API Community Manager control panel, click Community Builder.
+. Click *Settings* > *Advanced*.
+. In the section *Head Markup* click *Edit Head Markup*.
+. Insert this code:
++
+[source,css]
+----
+<style>
+    api-navigation {
+      --api-navigation-list-item-word-break: keep-all !important;
+    }
+</style>
+----
+. Optionally, replace `api-navigation` with any selector that contains the component `api-navigation`.
+. Optionally, replace `keep-all` with any valid value for the `word-break` CSS property.
++
+The default behavior is `break-word`.
```

Comment:
I don't understand what is going on here: is "keep-all" the default or "break-word," or are these not related?
This might be just ME misunderstands, so would you reread for audience clarity the relationships among...:

- The end of this task
- The statement about "break-word" (related or not to final optional step?)
- The change (?) to linking considerations (are these supposed to be related to the previous task?)

Reply:
"keep-all" and "break-word" are related: both are valid options. The example code used "keep-all", but I have changed it to the default, "break-word", for better clarity. I also updated the text for better clarity.

The linking considerations are another feature of the API Console component, but they are not otherwise related to the previous task.